net/bind/Makefile

```diff
@@ -52,6 +52,7 @@ define Package/bind/Default
   SECTION:=net
   CATEGORY:=Network
   DEPENDS:=+bind-libs +@OPENSSL_WITH_EC
+  EXTRA_DEPENDS:=bind-libs (=$(PKG_VERSION)-r$(PKG_RELEASE))
```

Comment:
Does this need to be in `EXTRA_DEPENDS`, or can the version constraint be added in `DEPENDS`?